src/utils/responseUtils.ts

```diff
@@ -2,4 +2,12 @@
  * Response status codes for responses that cannot have body.
  * @see https://fetch.spec.whatwg.org/#statuses
  */
-export const responseStatusCodesWithoutBody = [204, 205, 304]
+export const RESPONSE_STATUS_CODES_WITHOUT_BODY = [101, 103, 204, 205, 304]
```

Comment:
super duper nitty:

Should we make this a set instead of an array? super minimal but a bit more semantic

Reply:
CI seems to suggest we throw on 101/103, which I think makes sense.

101 is switching protocols (changing connection to http/2 or sockets)
103 is early hints

I'm not sure how interceptors handle those now if we received them from somewhere. If we ignore them  or what, but maybe we only need to remove them from here as they're not standard request/response style headers that would happen here?

Reply:
@mattcosta7, made the status codes a set. Adjusted the test.